app.py

```diff
@@ -30,7 +30,24 @@ def init_app(self):
         Note, this app doesn't register any commands at the moment as all it's functionality is
         provided through it's API.
         """
-        pass
+
+        app = self.import_module("tk_multi_reviewsubmission")
+
+        display_name = self.get_setting("display_name")
+
+        menu_caption = "%s..." % display_name
+        menu_options = {
+            "short_name": "send_for_review",
+            "description": "Send a version for review Using Shotgun Create",
```

Comment:
Seems a bit look for a menu item name. I'm sure our UX peeps will want to chime in. Now sure how to @ them here...

Reply:
This is not the display name, it's the description. Default is `Send for review` and renamed to  `Playblast to Shotgun Create` in Maya.